src/commands/synthetics/utils.ts

```diff
@@ -93,6 +83,32 @@ export const handleConfig = (test: Test, publicId: string, config?: ConfigOverri
   return handledConfig
 }
 
+const parseUrlVariables = (url: string, write: Writable['write']) => {
+  const context: TemplateContext = {
+    ...process.env,
+    URL: url,
+  }
+  try {
+    const objUrl = new URL(url)
+    const subdomainMatch = objUrl.hostname.match(SUBDOMAIN_REGEX)
+    const domain = subdomainMatch ? objUrl.hostname.replace(`${subdomainMatch[1]}.`, '') : objUrl.hostname
+
+    context.DOMAIN = domain
+    context.HOST = objUrl.host
+    context.HOSTNAME = objUrl.hostname
+    context.ORIGIN = objUrl.origin
+    context.PARAMS = objUrl.search
+    context.PATHNAME = objUrl.pathname
+    context.PORT = objUrl.port
+    context.PROTOCOL = objUrl.protocol
+    context.SUBDOMAIN = subdomainMatch ? subdomainMatch[1] : undefined
+  } catch {
+    write(`Unable to parse start url: ${url}`)
```

Comment:
I'm not sure it will be very clear to the user. Maybe `The start url ${url} contains variables so local overrides will be ignored`. Not really about it either because I don't think the user will understand what `local overrides` are so don't hesitate to find something better!